payment-service/src/main/java/com/hedvig/paymentservice/serviceIntergration/productPricing/dto/MarketInfo.kt

```diff
@@ -0,0 +1,10 @@
+package com.hedvig.paymentservice.serviceIntergration.productPricing.dto
+
+import javax.money.CurrencyUnit
+
+data class MarketInfo(
```

Comment:
We do not want to confuse market as a set in stone concept, this one is related to the contract

```suggestion
data class ContractMarketInfo(
```